site/docs/samples/cleanup.md

```diff
@@ -11,3 +11,17 @@ When you're finished experimenting with a sample, you may clean up as follows:
     ```bash
     NS="fybrik-system"; kubectl -n $NS get configmap | awk '/sample/{print $1}' | xargs  kubectl delete -n $NS configmap
     ```
+
+## Cleaning up the OpenMetadata data catalog
+=== "Kubernetes"
+    ```bash
+    helm delete openmetadata openmetadata-dependencies -n open-metadata

```

Comment:
please add that if you changed the default OMD installation namespace, you have to update the commands below.